server/routes/snippets.js

```diff
@@ -13,6 +13,25 @@ router.get('/', function (req, res) {
   });
 });
 
+router.get('/search', function (req, res) {
+  var options = {};
+  if (req.query.name) {
+    options.where = { name: req.query.name };
+  }
+  models.Snippet.findAll(options).then(function (snippets) {
```

Comment:
@vrael560 you are loading snippets without content and versions. Please use ```withVersions``` scope and serialize objects with ```toJson```